### ISSUE TITLE
Feature to deselect individual items in preview results

### ISSUE THREAD

**Issue by bendover22** [bleachbit/wishlist#3]
I don't see a way to remove / protect individual items from preview results from being cleaned, other than uncheck a whole category and re-run entire preview.
Unchecking an entire category isn't always desirable.

Is it possible to add ability to right click a preview results item & "remove from cleaning list" or "don't clean this item," etc?  Lots of apps have this sort of feature.  That might be easier to implement than adding check boxes beside each preview item to select / deselect items.

I imagine it's very common to want to protect 1 or 2 items in category results.

If items are deselected / protected, they could be grayed out and / or italicized or change background color to show they're protected from cleaning.  

Some apps use check marks, 'X's' to indicate selected and deselected list items, but that may be more work than graying out, italicizing, etc.

Note:  green & red colors, alone, are difficult for color blind persons to see.

**Comment by gqqnbig** [comment]
Any progress on this issue? 

I love bleachbit being open source, but the way it selects/deselects fies is bizarre. I expect a checkbox list where I want uncheck files to be deleted.

**Comment by az0** [comment]
You're right that this and (other GUI improvements) would be helpful, but before many GUI changes, though, a major problem to solve is modernizing the Python+GTK frameworks on Windows.  Thank you for your patience.

For now, I will mark this with a high priority label.

**Comment by az0** [comment]
The [next gen GUI](https://github.com/bleachbit/bleachbit_gui_next_gen), which is just a front-end demo, has a feature that works like this: select one or more files during preview (or clean), and then select the option to skip it. The status bar confirms it's added to the whitelist

There is a demo video below of the next gen gui.

https://github.com/user-attachments/assets/a7110366-3099-4753-ad14-0a63b84faf13



Question: would it be helpful if in the preferences, you could paste paths from the clipboard or drag and drop to add them to the whitelist (allowlist)?

**Comment by gqqnbig** [comment]
I have two suggestions based on this video recording.

1. When you right-click an item, the context menu should have "ignore this time" in addition to "Always ignore". 

The reason is that, I sometimes clean up batch by batch. If I've got enough free space, I stop deleting high-risk items.

2. When an item is ignored temporarily or permanently, this item should disappear from the list. After all, how do I know whether an item has been ignored because of my memory of a goldfish?

**Comment by gqqnbig** [comment]
> Question: would it be helpful if in the preferences, you could paste paths from the clipboard or drag and drop to add them to the whitelist (allowlist)?

I didn't actually try the prototype, but I guess pasting paths should be helpful. Drag and drop is no big deal.
